test/SourceKit/DocSupport/doc_swift_module1.swift

```diff
@@ -1,4 +1,4 @@
 // RUN: %empty-directory(%t.mod)
-// RUN: %swift -emit-module -o %t.mod/cake1.swiftmodule %S/Inputs/cake1.swift -disable-implicit-string-processing-module-import -parse-as-library
+// RUN: %swift -emit-module -o %t.mod/cake1.swiftmodule %S/Inputs/cake1.swift -disable-implicit-string-processing-module-import -disable-implicit-concurrency-module-import -parse-as-library
```

Comment:
These tests were subtly affected by the presence of the new module as well, so I just disabled overall implicit concurrency loading and modified the expected outputs accordingly. The tests are still exercising/verifying the functionality they were before.